src/frontend/pages/thank-you.tsx

```diff
@@ -1,5 +0,0 @@
-import { ReactElement } from 'react';
```

Comment:
Can you replace this with some kind of landing page for if they timeout of the waiting room?